### PR TITLE
Add improved Login screen

### PR DESCRIPTION
**Pull request by VictorVelarde**
An improved design for the main Login page, including small adjustments for responsiveness

image

### CODE REVIEW

File: template-skeleton/template/src/components/views/login/Login.js [hunk 1]
```diff
@@ -1,82 +1,137 @@
...
 
 const useStyles = makeStyles((theme) => ({
-  containerWrapper: {
-    flex: '1 1 auto',
-    display: 'flex',
+  fullContainer: {
+    backgroundColor: theme.palette.primary.main,
+    height: '100%',
+    minHeight: 600,
+    minWidth: 400,
```
Comment (neokore) [inline template-skeleton/template/src/components/views/login/Login.js:19]: Maybe we need to add some CSS rules for mobile

File: template-sample-app/template/src/components/views/login/Login.js [hunk 2]
```diff
@@ -1,77 +1,162 @@
...
 
 const useStyles = makeStyles((theme) => ({
-  containerWrapper: {
-    flex: "1 1 auto",
-    display: "flex",
+  fullContainer: {
+    backgroundColor: theme.palette.primary.main,
+    height: '100%',
+    minHeight: 600,
+    minWidth: 400,
```
Comment (neokore) [inline template-sample-app/template/src/components/views/login/Login.js:26]: Maybe we need to add some CSS rules for mobile

**Comment by VictorVelarde** [comment]
Those min-width and height were not needed indeed. I have removed them

**Review by neokore — APPROVED** [review]
LHTM

**Pull request merged**